src/app/screens/sendBtc/recipientSelector.tsx

```diff
@@ -51,41 +38,44 @@ function RecipientSelector({
     setAddressIsValid(true);
   };
 
-  const inputFeedback = useMemo(() => {
+  interface InputFeedback {
+    variant: 'danger';
+    message: string;
+  }
```

Comment:
nit: prefer types over interfaces unless we are making use of interfacer functionality. Also, this should maybe live up in the file by the other types.